Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/SimpleBridgeGeneratorImpl.kt

```diff
@@ -61,6 +62,7 @@ class SimpleBridgeGeneratorImpl(
             BridgedType.ULONG -> "uint64_t"
             BridgedType.FLOAT -> "float"
             BridgedType.DOUBLE -> "double"
+            BridgedType.NATIVE_VECTOR -> "NOT USED???" // "float __attribute__ ((__vector_size__ (16)))"
```

Comment:
If it is actually not used, then this should throw an exception instead.

Reply:
Done. Dead code which can't be easily removed.